cves/CVE-2003-0132.yml

```diff
@@ -69,52 +73,47 @@ unit_tested:
     Were automated unit tests involved in this vulnerability?
     Was the original code unit tested, or not unit tested? Did the fix involve
     improving the automated tests?
-
     For the "code" answer below, look not only at the fix but the surrounding
     code near the fix and determine if and was there were unit tests involved
     for this module.
-
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer:
+  answer: There were not unit tests involved in this vulnerability. 
   code:
   fix:
 
 discovered:
   question: |
     How was this vulnerability discovered?
-
     Go to the bug report and read the conversation to find out how this was
     originally found. Answer in longform below in "answer", fill in the date in
     YYYY-MM-DD, and then determine if the vulnerability was found by a Google
     employee (you can tell from their email address). If it's clear that the
     vulenrability was discovered by a contest, fill in the name there.
-
     The "automated" flag can be true, false, or nil.
     The "google" flag can be true, false, or nil.
-
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
-  date:
-  automated:
-  google:
-  contest:
+  answer: This bug was originally discovered by an investigation of a denial of 
+  service attack. It was first disclosed to a third party site, iDEFENSE, who
+  then notified apache. 
+  date: 01/23/2003
+  automated: false
+  google: false
+  contest: false
 
 subsystem:
   question: |
     What subsystems was the mistake in?
-
     Look at the path of the source code files code that were fixed to get
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged.
-  answer:
+  answer: 
```

Comment:
Was the vulnerability not in any of the subsystems?

Reply:
Will add an answer here shortly